### PR TITLE
issue-11 Rename Builder#with(File cwd) and Builder#with(Map<String, S…

### PR DESCRIPTION
**Pull request by xjj59307**
…tring> env) to avoid name clashes

### CODE REVIEW

**Comment by dakusui** [comment]
Didn't we have tests for these methods?

File: src/main/java/com/github/dakusui/cmd/Cmd.java [hunk 1]
```diff
@@ -139,12 +139,12 @@ public Builder with(Shell shell) {
       return this;
     }
 
-    public Builder with(File cwd) {
+    public Builder cwd(File cwd) {
```
Comment (dakusui) [inline src/main/java/com/github/dakusui/cmd/Cmd.java:142]: I think this is good.

File: src/main/java/com/github/dakusui/cmd/Cmd.java [hunk 2]
```diff
@@ -139,12 +139,12 @@ public Builder with(Shell shell) {
...
       this.cwd = cwd;
       return this;
     }
 
-    public Builder with(Map<String, String> env) {
+    public Builder env(Map<String, String> env) {
```
Comment (dakusui) [inline src/main/java/com/github/dakusui/cmd/Cmd.java:147]: I think this name is good.
But the method definition isn't cool enough. It's a separate issue, though.

**Comment by xjj59307** [comment]
@dakusui I've added a test case for Cmd Builder with all methods being refactored.

**Review by dakusui — COMMENTED** [review]
lgtm

**Comment by dakusui** [comment]
Cool, why don’t we merge it now?

File: src/main/java/com/github/dakusui/cmd/Cmd.java (hunk 2 above)
Comment (xjj59307) [reply src/main/java/com/github/dakusui/cmd/Cmd.java:147]: Thanks. Let's file another issue if you come up with a better idea.

**Pull request merged**